src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -1,3 +1,4 @@
+#include "Storages/PartitionCommands.h"
```

Comment:
```suggestion
#include <Storages/PartitionCommands.h>
```

Likely you need to update clangd to 20, that will include proper patch for this - https://github.com/llvm/llvm-project/pull/67749